instrument/event/event.go

```diff
@@ -5,7 +5,7 @@
 
 package event
 
-//go:generate stringer -type=Event
+//go:generate go run golang.org/x/tools/cmd/stringer -type=Event
```

Comment:
This is to stop assuming `stringer` is installed "globally" and instead pick it up from `go.mod` via `tools.go`.